week_2/init-s3.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
```

Comment:
o	Add versioning to S3 bucket (via AWS CLI commands)

Reply:
added